### ISSUE TITLE
Making the App Responsive and Look Good on Mobile Devices

### ISSUE THREAD

**Issue by MarcoMorawec** [sengngo/splurty#1]
On mobile things don't look as good as they could be.

One thing you could look into is to add the "mobile meta tag" to your app, so the responsive effect of Bootstrap actually shows up on Mobile.

Let me know in the comments once you figured that step out :)

**Comment by sengngo** [comment]
I believe the issue I had was this in app/assets/stylesheets/application.css.scss
I originally had the code as
// "bootstrap-sprockets" must be imported before "bootstrap" and "bootstrap/variables" 
@import "bootstrap-sprockets";
@import "bootstrap";

Since there was no space between the comment and @import it never imported bootstrap. I'm not really sure what // does though. However, it does seem to negate things in CSS.

**Comment by MarcoMorawec** [comment]
It doesn't look like your app looks better on Mobile yet.

Why don't you take a look into what the "mobile meta tag" does and see if you can figure it out that way :)

**Comment by sengngo** [comment]
Hi Marco,

I've tried to install the web development environment on my laptop so I
could work on it. I followed the instructions on
http://www.thefirehoseproject.com/cheat-sheets/working-on-existing-projects
and it seem to me that everything was working fine until I ran "$ rails
server". I'm getting the an error (i've attached a screen shot)

Can you please let me know what I'm doing wrong?

Seng

**Comment by sengngo** [comment]
Hi Marco,

To master.css.scss I added 
@-ms-view{
width: device-width;
}

and to application.html.erb I added 

<<meta name="viewport" content="width=device-width, initial-scale=1">

My reasoning is - this code should tell the browser to render to the width of the device screen rather than "zoom out" to the default. It was also recommend that I add this to CSS as well. 

I tested it on both a Samsung Note 4 and an Iphone 5 but no tablet (I don't have one at the moment.
